packages/react/test/specs/components/Provider/Provider-test.tsx

```diff
@@ -184,4 +187,46 @@ describe('Provider', () => {
       })
     })
   })
+
+  describe('calls provided renderer', () => {
+    test('calls renderFont', () => {
+      const theme: ThemeInput = {
+        fontFaces: teams.fontFaces,
+      }
+
+      const renderFontCalled = jest.fn()
+      const rendererMock = new RendererMock(renderFontCalled, jest.fn())
```

Comment:
https://jestjs.io/docs/en/jest-object.html#jestspyonobject-methodname

Have you tried `spyOn` instead?